### ISSUE TITLE
Enable wiki by default

### ISSUE THREAD

**Issue by 130s** [ros-infrastructure/bloom#416]
`wiki.ros.org` IMO has been and will be serving as the best central location for indexing/listing/browsing ROS packages existing out there. A key for a package to be listed there is to have its wiki page on the site. Documenting is even better (eg. as recommended by Tully in [a mailinglist thread](http://lists.ros.org/pipermail/ros-users/2013-December/068075.html)).

Right now, however, creating the package's wiki page is only noncompulsory, and requires manual operation. I just added a [wiki instruction for doing so](http://wiki.ros.org/rosdistro/Tutorials/Indexing%20Your%20ROS%20Repository%20for%20Documentation%20Generation#Create_a_wiki_page_for_your_package_in_very_short_steps). I presume that newer users might not even think of the fact that creating wiki page is very beneficial for them and for the community (for that I just also added [this](http://wiki.ros.org/action/info/Get%20Involved?action=diff&rev2=90&rev1=89)).

If running `bloom` automatically generates a wiki for the package it just handled, or at least recommends doing so by showing a possible URL (e.g. `http://wiki.ros.org/your_justreleased_pkg`), that hopefully will help reducing packages whose wiki pages are not yet created.

**Comment by asmodehn** [comment]
I also think that creating the wiki page automatically is a good idea.

However I am not convinced bloom would be the best place to do that...
Maybe it should be a job left for the buildfarm, when building that package ?

Or maybe whenever the shadow-fixed repository is made public, all wiki pages should be automatically updating, to stay uptodate with the packages that are available for everyone ?

**Comment by 130s** [comment]
> However I am not convinced bloom would be the best place to do that...
> Maybe it should be a job left for the buildfarm, when building that package ?

Yeah I thought about it. The reason why I thought `bloom` for this is because the wiki page may need filled with some info that requires manual input anyway, such as link to tutorials, documentation if the package creators/maintainers choose ros wiki as a doc location, etc. So to remind them, I thought the moment they run `bloom` might be a reasonable timing.
